client/components/action-panel/figure.jsx

```diff
@@ -1,9 +1,8 @@
 import clsx from 'clsx';
 import PropTypes from 'prop-types';
 
-const ActionPanelFigure = ( { inlineBodyText, align, children } ) => {
-	const figureClasses = clsx( {
-		'action-panel__figure': true,
+const ActionPanelFigure = ( { inlineBodyText = false, align = 'left', children } ) => {
```

Comment:
Hmm, with this classname logic:

```
[ `align-${ 'left' === align ? 'left' : 'right' }` ]: true,
```

Doesn't this mean that `right` was the default instead?

Reply:
Oops, fixed!